bedrock/firefox/templates/firefox/family/includes/modules/bullying.html

```diff
@@ -17,9 +17,8 @@ <h3 class="c-subtitle">Ugh, trolls amiright?</h3>
       </div>
 
       {% call browser_border() %}
-        <img src="{{ static('img/firefox/family/nested-hearts.svg') }}" alt="Nested hearts in a rainbow of colors">
-        <!-- might be easier to line up with flex and inline svgs -->
-        <div class="c-reactions"><span class="c-likes">12</span><span class="c-users">27</span><span class="c-comments">6</span></div>
+        <img src="{{ static('img/firefox/family/nested-hearts.svg') }}" alt="Nested hearts in a rainbow of colors" width="287" height="276">
+        <img src="{{ static('img/firefox/family/reaction-box.svg') }}" alt="12 Likes, 27 Friends, 6 Comments" width="328" height="104" class="c-reaction-box">
```

Comment:
I experimented with keeping the reaction box more of an HTML/CSS construction but the Fira Mono font was too different and we don't have any plans to animate this box, so there seemed little gain in avoiding the image at this point.